### DISCUSSION TITLE
军训晒黑记录(军训结束啦！)

### DISCUSSION THREAD

**Discussion by jsishsjsj** [share121/inter-knot#4543] [灌水]
![](https://th.bing.com/th/id/OIP.DYUEfWNetjLTlUcv7QgmMwHaHa?dpr=3&pid=ImgDetMain)

**补录**

今日洗澡后感觉没有晒黑）

![IMG_20240913_124653.jpg](https://github.com/user-attachments/assets/9a41a5b8-99b3-4c3e-8e4f-c7f70bddd8d2)

---

**第七天，军训结束啦！**

![IMG_20240912_174102.jpg](https://github.com/user-attachments/assets/8b5e11b6-fa46-4a96-80df-a7d6e6252c66)

黑了一点，总体变化不大。

**不过大家还是要买防晒的，防止晒伤。**

---

**虽然晒黑了一点，但没有分色**

![Image_1726068814693](https://github.com/user-attachments/assets/7b91c3aa-6d6b-497f-8c7c-aab5e66d52b7)

**第六天**

![IMG_20240911_184724.jpg](https://github.com/user-attachments/assets/a272da96-754f-409a-952a-ee72e6b05d37)

**第五天**

![IMG_20240910_190331.jpg](https://github.com/user-attachments/assets/fbf2a7b1-94cc-4549-9ea4-4b995a177fcf)

**第四天**

![IMG_20240909_185734.jpg](https://github.com/user-attachments/assets/d9a01b64-65dd-4b1c-b1fc-fee67dcd47bd)

**第三天**

![IMG_20240908_182524.jpg](https://github.com/user-attachments/assets/8d4fcd94-a265-4e0c-acc4-700647163fa4)

---

今天是第二天（不涂防晒霜，照片无美颜）
![IMG_20240907_175545](https://github.com/user-attachments/assets/bdffbb5d-f0c9-4566-a626-3b7baf9dd789)

**Comment by Signedinn** [comment]
兄弟你好香
![IMG_4264](https://github.com/user-attachments/assets/decdc224-ea37-4708-9836-44587423e6ef)

**Comment by Allium0** [comment]
我们这边今年军训都不热，可恶，为什么我那时候那么热😭

**Comment by sensei-is-playing-GenShin** [comment]
![微信截图_20240908231931](https://github.com/user-attachments/assets/c57a6113-da9b-42cc-b0fb-8098f558dc89)
在这晒的对吗？（doge）

**Reply by jsishsjsj** [reply]
那包的啊

**Reply by Robin-txdy** [reply]
晒得黢黑

**Comment by Maridizk** [comment]
兄弟你好香

**Comment by lllliufeng** [comment]
海南：

**Comment by ConsistDust3838** [comment]
同军训人枯了

**Comment by March-777th** [comment]
我们这边连着好几天没太阳了<img src="https://img-static.mihoyo.com/communityweb/upload/8f9225dcc57a028231f1d67fdab9135c.png" alt="三月七_骄傲" width="75" >

**Comment by fqing8485** [comment]
感觉没啥区别，再蹲几天

**Reply by jsishsjsj** [reply]
第六天已经黑了（裂开）

**Reply by ff688** [reply]
看出来了，记得做好防晒

**Reply by jsishsjsj** [reply]
> 看出来了，记得做好防晒

至少没有分色

**Comment by Robin-txdy** [comment]
我们这边超级热，还好我们不用军训，就是苦了那群新生了

**Comment by tingyu919** [comment]
兄弟你香香的

**Comment by Darkwind913** [comment]
兄弟别回头，有急事

**Comment by Junjun280** [comment]
好看，爱看<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Comment by 0xSelenicDove** [comment]
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by March-777th** [comment]
看饿了<img src="https://img-static.mihoyo.com/communityweb/upload/7bd889870907d2d96673d09afb53cf63.png" alt="三月七_哭" width="75" >

**Comment by arlcius** [comment]
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/34a86cbdf30331a9694cb9165188e3b5_6588321211650828715.png" alt="白露 饿" width="75" >看饿了
